docs/docs/user-guide/appendix/FAQ.md

```diff
@@ -0,0 +1,51 @@
+# Frequently Asked Questions
+
+### Is BioNeMo free to use?
+
+Yes, BioNeMo is free to use. BioNeMo code is licensed under the Apache 2.0 License. The Apache 2.0 License is a
+permissive open-source license that allows users to freely use, modify, and distribute software. With this license,
+users have the right to use the software for any purpose, including commercial use, without requiring royalties or
+attribution. Overall, our choice of the Apache 2.0 License allows for wide adoption and use of BioNeMo, while also
+providing a high degree of freedom and flexibility for users.
+
+### How do I install BioNeMo?
+
+BioNeMo is distributed as a Docker container through NVIDIA NGC. To download the pre-built Docker container and data
+assets, you will need a free NVIDIA NGC account.
+
+Alternatively, you can install individual sub-packages from within BioNeMo by following the corresponding README pages
+the [BioNeMo GitHub](https://github.com/NVIDIA/bionemo-framework).
```

Comment:
I worry about this, we really haven't tested our code or the install outside of a docker container. Things like apex and TE are tricky to build correctly, and I don't think we really have those as part of our install instructions.

Is it was possible to soften the expectation that this will work without some extra effort on their part?

Reply:
Added as part of @ntadimeti's suggestions below.